guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -36,6 +36,12 @@ ifdef::foreman-el,foreman-deb[]
 If required, you can disable Puppet agent on {ProjectServer} using the `--puppet-runmode=none` option.
 endif::[]
 
+ifdef::katello,satellite,orcharhino[]
+.Prerequisites
+
+* If you want to use an external PostgreSQL database, PostgreSQL must be installed on your host. For instructions, see xref:using-external-databases[Using external databases with {Project}].
```

Comment:
```suggestion
* If you want to use an external PostgreSQL database, PostgreSQL must be installed on your host.
For more information, see xref:using-external-databases[].
```

suggestion: split line by sentence to align to our "one sentence per line" convention.

FYI: If we use xrefs, we omit the title. This is handled by asciidoctor itself & simplifies maintenance.

Reply:
Another suggestion: Maybe reword "on your host"? I wonder if people could confuse this with their EL9 host for their Foreman/Katello instance. Maybe "external database host"?